src/services/RedisService.ts

```diff
@@ -0,0 +1,27 @@
+import { RedisClient } from 'redis';
+import { injectable, inject } from 'inversify';
+import { TYPES } from '../Types';
+import { IMap } from '../utils/Types';
+import { getAsync, setAsync, setObjectAsync, getObjectAsync } from '../utils/RedisUtils';
```

Comment:
I suggest we move it into RedisService. It brings no value outside RedisService and for that reason should not be as public API.

Reply:
The point was to segregate the implementation of the promises away from the controller to make it more readable

Reply:
I do understand however turning callbacks into promises is delegated to helper method anyway making it one liner. I do not see much of an issue having `promisify(client.get).bind(client)` in implementation of service methods. In my mind it makes more sense than making helpers methods available for potential reuse.